packages/mendel-config/src/bundle-config.js

```diff
@@ -7,6 +7,19 @@ function BundleConfig(id, options, config) {
     this.generator = options.generator || 'default';
 
     const {outdir} = config.baseConfig;
+    const supportedOptions = [
+        'entries',
+        'external',
+        'from',
+        'generator',
+        'manifest',
+        'outfile',
+        'outlet',
+        'require',
+        'runtime',
+    ];
```

Comment:
> The code you are referring to is done on first level keys.

`this.options` is updated in this (line https://github.com/yahoo/mendel/pull/90/files#diff-1f6220fbe492ebe47f6ff935346291aeL22).

After this, `this.options` will only have config defined for given bundle.